frontend/packages/dev-console/locales/en/devconsole.json

```diff
@@ -374,6 +374,7 @@
   "Sample repository:": "Sample repository:",
   "The Devfile in your Git repository is invalid.": "The Devfile in your Git repository is invalid.",
   "Validated": "Validated",
+  "Could not get Devfile for an unknown Git type": "Could not get Devfile for an unknown Git type",
```

Comment:
do we need to check on the wording with UXD/PM?

cc @jerolimov

Reply:
Sure! Please check with @serenamarie125 or @beaumorley